src/frontend/src/app/app.component.html

```diff
@@ -1 +1,2 @@
-<router-outlet></router-outlet>
+<app-header></app-header>
```

Comment:
This piece was handled by the `LayoutComponent` so I'm wondering what we're gaining by removing it.